docs/continuous_integration.md

```diff
@@ -30,12 +40,12 @@ shared code. As a result, any changes to this shared code must be made in the
 [TensorFlow repo](https://github.com/tensorflow/tensorflow) which will then automatically sync'd via a scheduled
 [GitHub workflow](../.github/workflows/sync.yml).
 # Merge Queue Details
-This section is probably only of interest if you plan to be doing surgery on the CI system. 
+This section is probably only of interest if you plan to be doing surgery on the CI system.
 ## Mergify
 We use [Mergify](https://mergify.com/) for our merge queue. [The documentation](https://docs.mergify.com/) is reasonably straight forward.
 ## Config File
 Our [mergify.yml](../.github/mergify.yml) is fairly standard. It triggers on `ci:ready_to_merge` label and requires all branch protection checks to pass before a PR can be added to the queue. When the PR is merged it removes the label.
 ## `ci:run` In Queue
 The one slightly complicated wrinkle in our system is the test suite being run only when the `ci:run` label is applied. As soon as the tests are run, the label is [removed](../.github/workflows/remove-labels.yml).
 
-As the queue processes each PR, it creates a temporary branch and merges in the results of the previous passing PRs my merging main. This merge resets all the test results, so the `ci:run` tag must be [reapplied](../.github/workflows/apply_cirun.yml) in queue.
\ No newline at end of file
+As the queue processes each PR, it creates a temporary branch and merges in the results of the previous passing PRs my merging main. This merge resets all the test results, so the `ci:run` tag must be [reapplied](../.github/workflows/apply_cirun.yml) in queue.
```

Comment:
What exactly is the difference between line 41 in the original file and line 51 here? Was it caused by CR/LF differences? Can't really see here because GitHub editor does not seem to highlight anything here.